modules/op-using-custom-pipeline-template-for-git-import.adoc

```diff
@@ -1,93 +0,0 @@
-// This module is included in the following assembly:
-//
-// *openshift-docs/cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc
-
-:_content-type: PROCEDURE
-[id="using-custom-pipeline-template-for-git-import_{context}"]
-= Using a custom pipeline template for creating and deploying an application from a Git repository
-
-As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
-
-[NOTE]
-====
-This feature is unavailable in {pipelines-title} 1.4 and earlier versions.
-====
-
-.Prerequisites
-
-Ensure that the {pipelines-title} 1.5 or later is installed and available in all namespaces.
-
-.Procedure
-
-. Log in to the {product-title} web console as a cluster administrator.
-. In the *Administrator* perspective, use the left navigation panel to go to the *Pipelines* section.
-.. From the *Project* drop-down, select the *openshift* project. This ensures that the subsequent steps are performed in the `openshift` namespace.
-.. From the list of available pipelines, select a pipeline that is appropriate for building and deploying your application. For example, if your application requires a `node.js` runtime environment, select the *s2i-nodejs* pipeline.
-+
-[NOTE]
-====
-Do not edit the default pipeline template. It may become incompatible with the UI and the back-end.
-====
-+
-.. Under the *YAML* tab of the selected pipeline, click *Download* and save the YAML file to your local machine. If your custom configuration file fails, you can use this copy to restore a working configuration.
-
-. Disable (delete) the default pipeline templates:
```

Comment:
IMHO lines 34 to 65 are very valuable, the new version doesn't say anything about removing/disabling the templates that are created by the operator.

Reply:
@ppitonak could you please explain in what use cases it is useful to remove/disable the templates that are created by the operator? Thanks!

Reply:
Per the discussion https://redhat-internal.slack.com/archives/CHG0KRB7G/p1690564699338199?thread_ts=1690217277.713569&cid=CHG0KRB7G - pretty sure that **Tekton Configuration** tab doesn't exist anymore in the Operator (at least it didn't when I tried this on a cluster), or else there's a step missing somewhere that makes it available. This being seemingly outdated was the reason I suggested removing the module.
Also, wouldn't the Operator just recreate these templates if there was an update or something? I might be wrong, I've just seen instances in the past where you didn't modify certain configs because the Operator would just override them anyway.

Reply:
You are right that UI for Tekton Config doesn't exist anymore but customers can still edit it using `oc edit tektonconfig config` or in UI `Administration -> CustomResourceDefinitions -> TektonConfig -> Instances -> config`

Reply:
Per @mramendi 's comment
> @ppitonak could you please explain in what use cases it is useful to remove/disable the templates that are created by the operator? Thanks!

It seems like there still isn't an answer for this. This PR seems to be blocked by something that I think could maybe be a separate discussion, i.e. if removing or disabling these isn't crucial in the scheme of things wrt admins creating templates, maybe it should be removed for now and revisited if it is actually requested by users or needed later?
Right now it seems like it's something that's just an option or a preference as opposed to something that's required, and I would like to get this PR closed since it has been open for weeks now.

Reply:
I don't know why would they disable default templates. You are right that this can be solved separately, not a blocker for this PR.